lib/osf-components/addon/components/file-metadata-manager/component.ts

```diff
@@ -181,14 +181,20 @@ export default class FileMetadataManagerComponent extends Component<Args> {
         this.inEditMode = true;
     }
 
+    @action
+    rollback() {
+        this.changeset.rollback();
+    }
+
```

Comment:
I don't think you need this. Generally, it is not a bad idea to define a `rollback()` action, provided that the action is invoked somewhere in the template or will be passed to some other components for invocation. But I don't think this is solving any issues. This is equivalent to doing `this.changeset.rollback()` on line 197.